docs/running_js_scripts.md

```diff
@@ -1,13 +1,44 @@
-Running JS Scripts in Remix
+Running JS Scripts
 ===========================
 
-Remix accepts async/await scripts to run [web3.js](https://web3js.readthedocs.io/)  or [ethers.js](https://docs.ethers.io/) commands. The script needs to be wrapped in a self executing function.
+_JavaScript (JS) is a lightweight, interpreted, or just-in-time compiled programming language with first-class functions._
```

Comment:
I"m not sure this helps - everyone knows what JS is - or is there another reason you are putting this in?

Reply:
This doesn't seem an issue. I would like to keep it.